src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -23,34 +23,55 @@ export const V1_DEVCONTAINER_FEATURES_FILE_NAME = 'devcontainer-features.json';
 // v2
 export const DEVCONTAINER_FEATURE_FILE_NAME = 'devcontainer-feature.json';
 
-export interface Feature {
+export type Feature = SchemaFeatureBaseProperties & SchemaFeatureLifecycleHooks & DeprecatedSchemaFeatureProperties & InternalFeatureProperties;
+
+export const CONTAINER_BUILD_FEATURES_DIR = '/opt/build-features';
```

Comment:
Would `/usr/share/devcontainer/features` be a better match? (Looking at https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard.)

Reply:
Hm ok that sounds good.  I also looked at that list but ended up choosing `opt` as it was short and somewhat related - but reading /usr/share that makes more sense 👍